pandas/io/json/_json.py

```diff
@@ -306,6 +306,7 @@ def read_json(
     precise_float: bool = False,
     date_unit=None,
     encoding=None,
+    errors: Optional[str] = "strict",
```

Comment:
IIRC  we discussed naming this `encoding_errors` or `errors_encoding` as `errors` is too general of an option name here.

Reply:
okay! `get_handle` already has an argument called `errors`. I assume the best would be to either use `encoding_errors` everywhere or everywhere but in `get_handle`?

Reply:
i think get_handle naming is just fine (its clear this applies to I/O). but in `read_csv/json`, e.g. the UX is where I would want it as more descriptive.